engine/lib/upgrades/2017032900-3.0.0_add_friends_acl-77c94ffca0a0a5d8.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Elgg 3.0.0 upgrade 2017032900
+ * add_friends_acl
+ *
+ * Introduces friends acls
+ */
+
+
+set_time_limit(0);
+$show_hidden = access_show_hidden_entities(true);
+
+$db = _elgg_services()->db;
+	
+// add subtype to access collections table
+$db->updateData("
+	ALTER TABLE {$db->prefix}access_collections
+	ADD COLUMN `subtype` varchar(255) NOT NULL DEFAULT '' AFTER `name`
+");
+
+// loop all users
+$users = elgg_get_entities([
```

Comment:
This upgrade can almost surely be done with 2 queries per user: 1 to create the ACL, 1 `SELECT ... INTO` to populate it. But like I recently mentioned, this PR yields identical data in two tables. Ideally we should combine them.